tests/unit/RequestTest.ts

```diff
@@ -1,8 +1,11 @@
-import * as Request from '../../src/libs/Request';
+import * as Request from '@src/libs/Request';
+import type {Middleware} from '@src/libs/Request';
+import type * as OnyxTypes from '@src/types/onyx';
 import * as TestHelper from '../utils/TestHelper';
 import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
 
 beforeAll(() => {
+    // @ts-expect-error TODO: Remove this once TestHelper (https://github.com/Expensify/App/issues/25318) is migrated to TypeScript.
```

Comment:
How about migrating TestHelper in this PR?

Reply:
I'm migrating TestHelper on another PR, I'll make sure this is removed before I merge the TestHelper one.